src/Domain/Insights/InsightFactory.php

```diff
@@ -127,13 +121,15 @@ private function runInsightCollector(OutputInterface $consoleOutput): void
     private function loadInsights(array $insights): array
     {
         $insightsAdded = [];
+        $cwd = getcwd();
```

Comment:
How does it handle the false return here?

Reply:
Same behavior as above :wink: I honestly never seen this function return false, did you ? In which case ?

Reply:
Never experienced it myself. According to the docs it says 
>On some Unix variants, getcwd() will return FALSE if any one of the parent directories does not have the readable or search mode set, even if the current directory does. See chmod() for more information on modes and permissions.

https://www.php.net/manual/en/function.getcwd.php